### ISSUE TITLE
Make mock data templating react to url/body/headers params

### ISSUE THREAD

**Issue by 255kb** [255kb/mockoon-feedback#4]
depends on #3

**Comment by eyalzf** [comment]
The title of the issue indicates url/body/headers params but in documentation I don't see any reference to body. Is there an helper to access body params? With specified path as parameter?

**Comment by 255kb** [comment]
@eyalzf no there is no helper for the body, title of this issue is misleading.

v1.0.0 is coming soon, I can try to implements this, considering that it could only work if body is valid JSON. I opened a new issue for this #47
